docs/guide/data.md

```diff
@@ -289,9 +229,18 @@ We can use `store.registerModule` to lazy-register this module in a route compon
 import fooStoreModule from '../store/modules/foo'
 
 export default {
-  asyncData ({ store }) {
-    store.registerModule('foo', fooStoreModule)
-    return store.dispatch('foo/inc')
+  computed: {
+    fooCount () {
+      return this.$store.state.foo.count
+    }
+  },
+
+  ssrPrefetch () {
+    return this.fooInc()
+  },
+
+  mounted () {
+    this.fooInc()
```

Comment:
Why not checking here also if `this.fooCount` is not set @Akryum ?

Reply:
Fixed, and more improvements ✔️